cdp_scrapers/instances/seattle.py

```diff
@@ -77,6 +77,7 @@ def __init__(self):
                 "Pursuant to Washington State",
             ],
             known_persons=known_persons,
+            person_aliases={"Dan Strauss": set(["Daniel Strauss"])},
         )
```

Comment:
I think it's better to  store `{"Dan Strauss": set(["Daniel Strauss"])}` as a constant like `known_persons`. It's easier to change without having to dig into the scraper code.

Reply:
:bulb: